src/factory.js

```diff
@@ -124,21 +133,25 @@ export default function factory(React, empty) {
           } else if(isAuthenticating) {
             return <LoadingComponent {...otherProps} />
           } else {
-            // Don't need to display anything because the user will be redirected
-            return React.createElement(empty)
+            // Unauthorized: Display FailureComponent or nothing
+            // if FailureComponent is undefined the user will never see
+            // this because they will be redirected to failureRedirectPath
+            return FailureComponent ? <FailureComponent authData={authData} {...otherProps} /> : React.createElement(empty)
```

Comment:
Am I missing something? I dont see a change in your last push unless github diff is being slow